src/webserver.cc

```diff
@@ -116,7 +115,7 @@ void Webserver::runThread(int threadIndex) {
             tcp::socket socket(io_service_);
 
             if ( !  acceptConnection(socket)) {
-                return;
+                return; // TODO: is this really what you want? one failed connection will crash your server.
```

Comment:
Thanks! I'm sure it won't be an issue but in principle you should look out for it.